src/desktop/src/ui/components/input/Seed.js

```diff
@@ -123,6 +123,13 @@ class SeedInput extends React.PureComponent {
         });
     };
 
+    onTextDrop = async (seed) => {
+        if (seed.length === MAX_SEED_LENGTH) {
```

Comment:
I think we should match how the validation has been done previously: https://github.com/iotaledger/trinity-wallet/blob/7500ce006b4da868aeeda54453f0ddffe1cea739/src/desktop/src/ui/components/input/Seed.js#L96